prefect_gcp/cloud_run.py

```diff
@@ -398,6 +407,30 @@ async def run(self, task_status: Optional[TaskStatus] = None):
             )
             return result
 
+    @sync_compatible
+    async def kill(self, identifier: str, grace_seconds: int = 30) -> None:
+        """
+        Kill a task running Cloud Run.
+
+        Args:
+            identifier: The Cloud Run Job name. This should match a
+                value yielded by CloudRunJob.run.
+        """
+        if grace_seconds != 30:
```

Comment:
Why 30 here? Is that the default wait time? Otherwise it might make sense to set the default to 0 indicate that it's going to delete immediately.

Reply:
What is Google's behavior here? The linked doc does not discuss termination signals.

Reply:
I thought it might have been the infrastructure block default; taken from https://github.com/PrefectHQ/prefect-aws/pull/163/files#diff-31c73194d9fd094cabf83de9a5de6fb5be3f781ad540ed40f1cd4da589c49e4bR561

Reply:
We aren't passing it through yet, but I can imagine the agent passing it via a setting in the future. 30s seems to be the standard everywhere here.

Reply:
Ah yeah - it looks like 30 seconds is the kubernetes default, which seems like a reasonable guess given that the grace period doesn't seem to be documented anywhere.